api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java

```diff
@@ -911,4 +919,12 @@ public String getOsDisplayName() {
     public String getPoolType() { return poolType; }
 
     public void setPoolType(String poolType) { this.poolType = poolType; }
+
+    public void setBytesReceived(final Long bytesReceived) {
```

Comment:
I don't fully understand why the key 'final' is being used here. You don't need to use it here.
am I wrong?